cmake/thirdparty/SetupSeracThirdParty.cmake

```diff
@@ -4,209 +4,212 @@
+    endif()
 
-#
-# Check for optional Axom headers that are required for Serac
-#
+    # MFEMSidreDataCollection.hpp
+    find_path(
+        _mfemdatacollection_found MFEMSidreDataCollection.hpp
+        PATHS ${AXOM_DIR}/include/axom/sidre/core
+        NO_DEFAULT_PATH
+        NO_CMAKE_ENVIRONMENT_PATH
+        NO_CMAKE_PATH
+        NO_SYSTEM_ENVIRONMENT_PATH
+        NO_CMAKE_SYSTEM_PATH
+    )
+    if (NOT _mfemdatacollection_found)
+        message(FATAL_ERROR "Given AXOM_DIR did not contain a required header: axom/sidre/core/MFEMSidreDataCollection.hpp"
+                            "\nTry building Axom with '-DAXOM_ENABLE_MFEM_SIDRE_DATACOLLECTION=ON'\n ")
+    endif()
 
-# sol.hpp
-find_path(
-    _sol_found sol.hpp
-    PATHS ${AXOM_DIR}/include/sol
-    NO_DEFAULT_PATH
-    NO_CMAKE_ENVIRONMENT_PATH
-    NO_CMAKE_PATH
-    NO_SYSTEM_ENVIRONMENT_PATH
-    NO_CMAKE_SYSTEM_PATH
-)
-if (NOT _sol_found)
-    message(FATAL_ERROR "Given AXOM_DIR did not contain a required header: sol/sol.hpp"
-                        "\nTry building Axom with '-DBLT_CXX_STD=c++14' or higher\n ")
-endif()
 
-# LuaReader.hpp
-find_path(
-    _luareader_found LuaReader.hpp
-    PATHS ${AXOM_DIR}/include/axom/inlet
-    NO_DEFAULT_PATH
-    NO_CMAKE_ENVIRONMENT_PATH
-    NO_CMAKE_PATH
-    NO_SYSTEM_ENVIRONMENT_PATH
-    NO_CMAKE_SYSTEM_PATH
-)
-if (NOT _luareader_found)
-    message(FATAL_ERROR "Given AXOM_DIR did not contain a required header: axom/inlet/LuaReader.hpp"
-                        "\nTry building Axom with '-DLUA_DIR=path/to/lua/install'\n ")
-endif()
+    #------------------------------------------------------------------------------
+    # MFEM
+    #------------------------------------------------------------------------------
+    include(${CMAKE_CURRENT_LIST_DIR}/FindMFEM.cmake)
 
-# MFEMSidreDataCollection.hpp
-find_path(
-    _mfemdatacollection_found MFEMSidreDataCollection.hpp
-    PATHS ${AXOM_DIR}/include/axom/sidre/core
-    NO_DEFAULT_PATH
-    NO_CMAKE_ENVIRONMENT_PATH
-    NO_CMAKE_PATH
-    NO_SYSTEM_ENVIRONMENT_PATH
-    NO_CMAKE_SYSTEM_PATH
-)
-if (NOT _mfemdatacollection_found)
-    message(FATAL_ERROR "Given AXOM_DIR did not contain a required header: axom/sidre/core/MFEMSidreDataCollection.hpp"
-                        "\nTry building Axom with '-DAXOM_ENABLE_MFEM_SIDRE_DATACOLLECTION=ON'\n ")
-endif()
 
+    #------------------------------------------------------------------------------
+    # Tribol
+    #------------------------------------------------------------------------------
+    if(TRIBOL_DIR)
+        serac_assert_is_directory(VARIABLE_NAME TRIBOL_DIR)
 
-#------------------------------------------------------------------------------
-# MFEM
-#------------------------------------------------------------------------------
-include(${CMAKE_CURRENT_LIST_DIR}/FindMFEM.cmake)
+        find_package(tribol REQUIRED
+                            NO_DEFAULT_PATH 
+                            PATHS ${TRIBOL_DIR}/lib/cmake)
```

Comment:
I think will set `tribol_FOUND` and not `TRIBOL_FOUND` - is that a problem?

Reply:
its definitely TRIBOL_FOUND (i just confirmed) and i also confirmed the tests ran since this could have failed silently by just not enabling the feature.